src/usdb_syncer/path_template.py

```diff
@@ -163,13 +163,16 @@ def evaluate(self, song: UsdbSong) -> str:
             case PathTemplatePlaceholder.TITLE:
                 return song.title
             case PathTemplatePlaceholder.GENRE:
-                return song.genre
+                first_genre, _, _ = song.genre.partition(",")
```

Comment:
I would prefer to use the existing accessors to avoid duplicating the splitting logic, e.g. `next(song.genres()) or ""`.

Reply:
Changes have been applied 👍  also made it up-to-date with latest main